base/float.jl

```diff
@@ -1,5 +1,7 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+const IEEEFloat = Union{Float16, Float32, Float64}
```

Comment:
I just moved this from `Base.Math`. I'll leave that kind of change to you :smile:. Should be pretty simple, though.